Procfile

```diff
@@ -0,0 +1 @@
+web: java -Dserver.port=$PORT $JAVA_OPTS -jar main.webapp/build/libs/main.webapp-0.1.0-SNAPSHOT.jar
```

Comment:
What this?

Reply:
This is the command used by Heroku to run the application. Since the jar that we actually want to execute is in main.webapp instead of in the root build folder like their defaults assume, I had to specify. I will try to get the hardcoded version number out of there

Reply:
If that's the case, why did we have to use one-jar at all? Couldnt we run whatever?

Reply:
well, yes but we'd have to have some task to copy all the libs somewhere. anyway I'm gonna switch to capsule because until Spring Boot 2.0 the bootRepackage task doesn't extend jar (Spring Boot 2.0 has bootJar and bootWar tasks that extend jar and war respectively, and I currently want the flexibility of extending jar)

Reply:
Ok. My point with that was mainly, if we just run something, why wouldn't we just docker it for consistenticy and known environment reasons?